src/python/WMCore/WMException.py

```diff
@@ -114,6 +116,13 @@ def addInfo(self, **data):
         exception instance
         """
         for key, value in viewitems(data):
+            # unicode sandwich: convert sequence of bytes strings to
+            # unicode codepoints strings as soon as possible
+            # assumption: value is not iterable (list, dict, tuple, ...)
+            if isinstance(key, bytes):
+                key = key.decode("utf-8")
```

Comment:
Should we add the `ignore` option here (and below)? Just so we can be sure that non-utf-8 compliant data won't break this?

Reply:
I see why you want to add this, I agree with you, thanks!